sdk/graphql-transport/src/methods.ts

```diff
@@ -810,6 +810,9 @@ export const RPC_METHODS: {
             referenceGasPrice: String(systemState.referenceGasPrice),
             safeMode: systemState.safeMode?.enabled!,
             safeModeComputationRewards: String(systemState.safeMode?.gasSummary?.computationCost),
+            safeModeComputationRewardsBurned: String(
+                systemState.safeMode?.gasSummary?.computationCostBurned,
+            ),
```

Comment:
```suggestion
            safeModeComputationCharges: String(systemState.safeMode?.gasSummary?.computationCost),
            safeModeComputationChargesBurned: String(
                systemState.safeMode?.gasSummary?.computationCostBurned,
            ),
```

Reply:
This suggestion will be outdated due to https://github.com/iotaledger/iota/pull/5270/commits/3d65d4a39724a007ab66be516b08e4bb0fc44c9d and https://github.com/iotaledger/iota/pull/5270/commits/3f5933339b60e7b381332c09e578516965fc801c

`IotaSystemStateSummaryV1` will now be kept unchanged with field `safe_mode_computation_rewards` and `IotaSystemStateSummaryV2` introduced with renamed and additional fields.